selene/src/main.rs

```diff
@@ -391,6 +391,55 @@ fn read_file(checker: &Checker<toml::value::Value>, filename: &Path) {
     );
 }
 
+/// Reads a config file and return content and path. None if not found or empty.
+fn read_config_file() -> Option<(String, Option<PathBuf>)> {
+    let config_paths_to_check = [".selene.toml", "selene.toml"];
+    let mut config_contents = String::new();
+    let mut config_path = None;
+
+    for path in &config_paths_to_check {
+        if let Ok(contents) = fs::read_to_string(path) {
+            config_contents = contents;
+            config_path = Some(PathBuf::from(path));
+            break;
+        }
+    }
+
+    if config_contents.is_empty() {
+        return None;
+    }

```

Comment:
Empty config should be handled separately from missing config.

Reply:
Solved in https://github.com/Kampfkarren/selene/pull/591/commits/ade463c78eca904e2d3fd801c3f6a792f258edc6